### ISSUE TITLE
V 2.0.0 built 3 - incorrect html syntax

### ISSUE THREAD

**Issue by birdmedia** [codefog/contao-cookiebar#42]
The new syntax **aria-describedby="cookiebar-desc"** requires an element with the id="cookiebar-desc" which is yet missing. The status quo is not W3C compliant.
